tools/hardware.yml

```diff
@@ -208,6 +208,14 @@ devices:
         kernel: PLIC_PPTR
         kernel_size: 0x04000000
 
+  # SiFive CLINT
+  - compatible:
+      - riscv,cpu-intc
```

Comment:
This is somewhat problematic as in fact all the currently supported RISC-V boards have `riscv,cpu-intc` despite having differently sized CLINTs. If there a current way in this hardware YAML to not specify the `kernel_size` as fixed?

Reply:
Right, so omitting it will work assuming the CLINT is always less than `1 << 21` in size. This seems like a reasonable assumption, but a very subtle one. In this case since all platforms have a CLINT that is less than `1 << 21`, specifying `kernel_size` doesn't actually do anything.

Reply:
Nice! Please do leave a comment for that if you're using it, though.

Reply:
Done